### PR TITLE
Ft get follower and following

### PR DESCRIPTION
**Pull request by verdotte**
#### What does this PR do?
get the list of followers and followings

#### Description of Task to be completed?
-implement retrieve list of followers of a given user
-implement retrieve list of following of a given user
-write test(s)

#### How should this be manually tested?
Run npm test

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

**Comment by verdotte** [comment]
[Get the user's following and followers](https://trello.com/c/xj23u9Q7/50-get-the-users-following-and-followers)

File: src/routes/api/v1/validators/profileValidator.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ const updateProfile = Joi.object().keys({
   bio: Joi.string()
     .trim()
     .min(20),
+  profession: Joi.string().max(50).min(2),
```
Comment (gracelungu) [inline src/routes/api/v1/validators/profileValidator.js:29]: @verdotte There is already a PR for this

File: src/routes/api/v1/follow.js [hunk 2]
```diff
@@ -10,5 +10,14 @@ router
   .all(asyncHandler(checkUsername))
   .post(asyncHandler(FollowController.follow))
   .delete(asyncHandler(FollowController.unFollow));
+  
+router
+  .route('/:username/follower')
+  .get(asyncHandler(checkUserFromParams), asyncHandler(FollowController.getFollowers));
+
+router
+  .route('/:username/following')
```
Comment (gracelungu) [inline src/routes/api/v1/follow.js]: @verdotte `/followers` and `/followings`

**Review by gracelungu — APPROVED** [review]

File: src/routes/api/v1/follow.js (hunk 2 above)
Comment (verdotte) [reply src/routes/api/v1/follow.js]: Noted

**Pull request merged**